framework/source/class/qx/ui/basic/Image.js

```diff
@@ -587,6 +587,14 @@ qx.Class.define("qx.ui.basic.Image",
         {
           var pixel = "px";
           var styles = {};
+          
+          //inherit styles from current element
+          var currentStyles = currentContentElement.getAllStyles();
+          if( (currentStyles!==null) && (currentStyles!==undefined) ) {
```

Comment:
Hi thanks for telling me.. I always had doubts with this .. it this exactly
the same behaviour? cheers!
On Feb 24, 2016 14:00, "Tobias Oetiker" <notifications@github.com> wrote:

> In framework/source/class/qx/ui/basic/Image.js
> <https://github.com/qooxdoo/qooxdoo/pull/176#discussion_r53934192>:
>
> > @@ -587,6 +587,14 @@ qx.Class.define("qx.ui.basic.Image",
> >          {
> >            var pixel = "px";
> >            var styles = {};
> > +
> > +          //inherit styles from current element
> > +          var currentStyles = currentContentElement.getAllStyles();
> > +          if( (currentStyles!==null) && (currentStyles!==undefined) ) {
>
> how about just a simple
>
> if (currentStyles) { ...
>
>
> https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/undefined
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qooxdoo/qooxdoo/pull/176/files#r53934192>.
>